src/Tribe/Integrations/Manager.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+
+/**
+ * Class Tribe__Events__Integrations__Manager
+ *
+ * Loads and manages the third-party plugins integration implementations.
+ */
+class Tribe__Events__Integrations__Manager {
+
+	/**
+	 * @var Tribe__Events__Integrations__Manager
+	 */
+	protected static $instance;
+
+	/**
+	 * The class singleton constructor.
+	 *
+	 * @return Tribe__Events__Integrations__Manager
+	 */
+	public static function instance() {
```

Comment:
Making the class constructor private makes it a pain to test; we are providing an ` instance` method for that reason. We have the same case repeating similarly throughout our code where the `__construct` method is public; this class has also an empty constructor so even instantiated twice it should cause no risks.